HW6/Dockerfile

```diff
@@ -14,16 +14,21 @@ RUN apt-get update
 RUN apt-get install -y mariadb-client
 
 # copy over code
-#COPY baseball.sql /baseball.sql
+COPY baseball.sql /data/baseball.sql
```

Comment:
maybe make a volume for the baseball.sql since its so big instead of copying it into the container